README.md

```diff
@@ -603,7 +603,7 @@ If you're using [Spring](https://github.com/rails/spring) to speed up test suite
 
     ```ruby
     # Disable Rails's static asset server (Apache or nginx will already do this)
-    config.serve_static_files = false
+    config.public_file_server.enabled = false
```

Comment:
I'm thinking we should document the different config for different versions of Rails.